docs/raw/serving-community-history.md

```diff
@@ -0,0 +1,315 @@
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Table of Contents](#table-of-contents)
+- [Terminology](#terminology)
+- [Supported platforms](#supported-platforms)
+- [Requirements / Assumptions](#requirements--assumptions)
+- [Overview](#overview)
+  - [Serving community history archives](#serving-community-history-archives)
+  - [Serving archives for missed messages](#serving-archives-for-missed-messages)
+  - [Receiving community history archives](#receiving-community-history-archives)
+- [Storing live messages](#storing-live-messages)
+- [Exporting messages for bundling](#exporting-messages-for-bundling)
+- [Message history archives](#message-history-archives)
+  - [WakuMessageHistoryArchive](#wakumessagehistoryarchive)
+- [Creating magnet links](#creating-magnet-links)
+- [Bundling history archives into archive indices](#bundling-history-archives-into-archive-indices)
+  - [WakuMessageArchiveMetadata](#wakumessagearchivemetadata)
+- [Message archive distribution](#message-archive-distribution)
+- [Canonical message histories](#canonical-message-histories)
+- [Fetching messsage archives](#fetching-message-archives)
+  - [Downloading message archive indices](#downloading-message-archive-indices)
+  - [Downloading individual archives](#downloadinng-individual-archives)
+- [Storing historical messages](#storing-historical-messages)
+- [Considerations](#considerations)
+  - [Bandwidth consumption](#bandwidth-consumption)
+  - [Multiple community owners](#multiple-community-owners)
+
+## Terminology
+
+The following terminology is used throughout this specification:
+
+| Name                 | References |
+| -------------------- | --- |
+| Waku node            | An Ethereum node with Waku V1 enabled, or a [10/WAKU2](https://rfc.vac.dev/spec/10/) node that implements [11/WAKU2-RELAY](https://rfc.vac.dev/spec/11/)|
+| Store node           | A Waku node that implements [11/WAKU-MAILSERVER](/spec/11) or [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) respectively |
+| Waku network         | A group of Waku nodes connected through the internet connection and forming a graph |
+| Community owner      | A Status user that owns a Status community |
+| Community member     | A Status user that is part of a Status community |
+| Community owner node | A Status node with message archive capabilities enabled, run by a community owner |
+| Community member node| A Status node with message archive capabilities enabled, run by a community member |
+| Live messages        | Waku messages received through the Waku network |
+
+
+## Supported platforms
+
+Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in desktop clients. Mobile clients SHOULD NOT implement this functionality due to bandwidth and storage limitations.
```

Comment:
@PascalPrecht @staheri14 one thing to note here: although supporting the Community History Archive Service on mobile is definitely outside the scope of the MVP, it *might* be something we wish to consider in the future.  The Community History Archive Service is less relevant on mobile because mobile users are more likely to be dipping into the latest conversation.

However in many countries in the world mobile bandwidth is both unlimited and extremely high speed (for example mobile data in Ukraine is as fast as an ADSL connection and unlimited data is around $5 per month), and these days smartphones have pretty powerful CPUs and 256GB to 512GB storage in a phone is no longer uncommon.  Given this, I think its definitely conservable that post-MVP we may wish to explore bringing the Community History Archive Service to Status Mobile, with perhaps the main difference relative to Status Desktop being that on Status Mobile it might be off by default.